### PR TITLE
Update docs for auth

### PR DESCRIPTION
**Pull request by TheIronDev**
Update docs for auth

### CODE REVIEW

**Review by colerogers — APPROVED** [review]

File: src/providers/auth.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const provider = 'google.firebase.auth';
 /** @hidden */
 export const service = 'firebaseauth.googleapis.com';
 
-/** Resource level options */
+/** @public Resource level options */
```
Comment (inlined) [inline src/providers/auth.ts:57]: visibility modifiers should be on their own line

File: src/providers/auth.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export interface UserOptions {
 }
 
 /**
- * Handle events related to Firebase authentication users.
+ * @public Handle events related to Firebase authentication users.
  */
 export function user(userOptions?: UserOptions) {
```
Comment (inlined) [inline src/providers/auth.ts:69]: Add param and returns comments

File: src/providers/auth.ts [hunk 3]
```diff
@@ -100,14 +100,14 @@ export class UserBuilder {
     private userOptions?: UserOptions
   ) {}
 
-  /** Respond to the creation of a Firebase Auth user. */
+  /** @public Respond to the creation of a Firebase Auth user. */
```
Comment (inlined) [inline src/providers/auth.ts:103]: "Responds" since we pretend an implicit "this method..."

Comment (TheIronDev) [reply src/providers/auth.ts]: Done

File: src/providers/auth.ts (hunk 2 above)
Comment (TheIronDev) [reply src/providers/auth.ts]: Done

File: src/providers/auth.ts (hunk 1 above)
Comment (TheIronDev) [reply src/providers/auth.ts]: Done

**Pull request merged**